crates/sdk/macros/src/logic.rs

```diff
@@ -42,7 +42,7 @@ impl<'a> TryFrom<LogicImplInput<'a>> for LogicImpl<'a> {
     type Error = Errors<'a, ItemImpl>;
 
     // TODO: This unwrap() call needs to be corrected to return an error.
-    #[allow(clippy::unwrap_in_result)]
+    #[expect(clippy::unwrap_in_result, reason = "TODO: This is temporary")]
```

Comment:
It's infallible

https://github.com/calimero-network/core/blob/d124aa3592c969d4aeb2879a46830d4ba1046ec4/crates/sdk/macros/src/sanitizer.rs#L251-L253

Reply:
Feel free to follow up on the areas like this that are flagged for you